src/mcode-pilot/components/TreatmentOptionsOutcomes/TreatmentOptionsOutcomes.jsx

```diff
@@ -173,14 +179,14 @@ export default class TreatmentOptionsOutcomes extends Component {
     renderRow(row, compareRow) {
         if (row.length === 0) return null;
 
-        const { name, totalPatients, sideEffects } = row;
+        const { name, displayName, totalPatients, sideEffects } = row;
```

Comment:
[DP3] `name` is no longer used as a variable. Do we need to create this variable in the first place anymore? It's defined first in the `generateRow` function but i'm not sure we need it -- if we need to conditionally set displayName based on the contents of treatments, we can do that I think without the separate variable treatmentName.

Reply:
I was being safe in case `name` was used elsewhere but I'll just remove `displayName` and turn `name` into `displayName`.